packages/@aws-cdk/aws-codebuild/test/test.codebuild.ts

```diff
@@ -9,6 +9,135 @@ import codebuild = require('../lib');
 
 export = {
   'default properties': {
+    'with build image'(test: Test) {
```

Comment:
How is this related?

Reply:
How did this get in? I didn't even git add this change o_O